guides/index.md

```diff
@@ -1,3 +1,39 @@
-This documentation is a work in progress and is supposed to be a very high level overview of the Cardstack architecture and how to use various parts of the system. These pages are not designed to be very detailed **API** documentation and may have some inconsistencies at first.
 
-There should be a pencil icon at the top right of every page that will allow you to edit this documentation so please feel free to contribute if you know how to make things a bit better!
+These work-in-progress documentation resources provide a high level overview of features, the project architecture, how to use Cardstack to power your own creations, and some pointers for developers who are working on the framework itself.
+
+## What is Cardstack?
+
+Cardstack is an open source framework that enables Web 3.0. “Cards” form the building blocks that allow for bringing in data from disparate sources, beautiful UI animations, encapsulated and flexible data schemas, and a quick ramp-up experience. Both JavaScript developers and end-users are empowered to add common application features (CMS, notifications, authentication).
```

Comment:
We should define Web 3.0 or at least link to a definition

Reply:
linked